pkg/cmd/roachtest/tests/row_level_ttl.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2023 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package tests
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster"
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/registry"
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/spec"
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test"
+	"github.com/cockroachdb/cockroach/pkg/util/timeutil"
+)
+
+func registerRowLevelTTLDuringTPCC(r registry.Registry) {
+	r.Add(makeRowLevelTTLDuringTPCC(r.MakeClusterSpec(7, spec.CPU(4)), 1500, 100, 30*time.Minute, false /* expiredRows */))
```

Comment:
Can you make vars for the shared args instead of copying and pasting them?

Reply:
done